101-workshop-nodejs/ex1-creating-and-invoking-actions/REVIEW.md

```diff
@@ -88,7 +88,7 @@ False. Actions can invoke other actions, acting as a proxy, using the NPM Apache
 (x) false
 
 [explanation]
-False. Using Promises in NodeJS, it is possible for your function to return while waiting for the result of the Promise.
```

Comment:
Promise needs to be capitalized.

Reply:
<img width="951" alt="Screen Shot 2020-04-08 at 11 35 49 AM" src="https://user-images.githubusercontent.com/26611298/78803589-3123c980-798d-11ea-8a07-08288f8ce845.png">

I was going by the style/edit from these three pieces of content on Developer where it isn't capitalized. But it's up to you!

Reply:
Can we capitalize and/or emphasize on first usage on page like we did for the components of the prog. model?